src/main/java/org/teamresistance/frc/Robot.java

```diff
@@ -1,13 +1,46 @@
 package org.teamresistance.frc;
 
 import edu.wpi.first.wpilibj.IterativeRobot;
+import org.strongback.Strongback;
 
 /**
  * Main robot class. Override methods from {@link IterativeRobot} to define behavior.
+ *
+ * @author Shreya Ravi
  */
 public class Robot extends IterativeRobot {
 
```

Comment:
Same here with the nitpick: I prefer not to have the blank line since the following line is an instance variable.